milabench/cli.py

```diff
@@ -199,6 +207,9 @@ def _get_multipack(
     now = str(datetime.today()).replace(" ", "_")
     run_name = run_name.format(time=now)
 
+    system_config = build_system_config(system_config_path)
+    overrides = merge(overrides, {"system": system_config})
+
     base_defaults = get_base_defaults(base=base, arch=deduce_arch(), run_name=run_name)
```

Comment:
Here we can use the `system.arch` instead

Reply:
I use a slightly different approach which is related to https://github.com/mila-iqia/milabench/pull/130#discussion_r1209438251